.github/scripts/debug_e2e_storage.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+docker run -v $(pwd)/docker/data:/data --network container:damian -e NODE_URL=127.0.0.1:9943 -e STORAGE_DEBUG=yes aleph-e2e-client:latest
```

Comment:
imho it would be much better to set environment variables in the pipeline itself then create different scripts for different sets

Reply:
Or use the same script but pass env as command line param?

Reply:
whatever

Reply:
not here anymore